docs/API_walkthrough.md

```diff
@@ -0,0 +1,644 @@
+# Quickstart
+
+This quickstart guide shows a real use case scenario of interaction with the API in an Apiculture context.
+
+## Prepare your environment
+
+The provided examples make use of the HTTPie command line tool (installation instructions: https://httpie.org/docs#installation)
+
+All requests are grouped in a Postman collection that can be found [here](https://github.com/stellio-hub/stellio-docs/blob/master/API_Quick_Start.postman_collection.json).
```

Comment:
I'm not sure this link is gonna work. Did you try it locally?

Reply:
I putted it in with Master (preparing the merge). I tested it locally (with the feature branch in the link) and it is taking me to the github page. It is like an external link.

Reply:
it is missing folder 'collection'